lite/CMakeLists.txt

```diff
@@ -145,12 +148,28 @@ if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/build_examples.cmake")
 endif()
 
 
-install(TARGETS ${XMP_INSTALLS}
-        RUNTIME DESTINATION "bin"
-        LIBRARY DESTINATION "lib"
-        ARCHIVE DESTINATION "lib"
-        INCLUDES DESTINATION "include")
+include(GNUInstallDirs)
 
+if(LIBM_REQUIRED)
+  set(LIBM "-lm")
+endif()
```

Comment:
I suggest to add this line into this place to make an order:
https://github.com/libxmp/libxmp/blob/master/cmake/libxmp-checks.cmake#L118

Reply:
Done this (will force-push shortly.)